mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/GEOR_custom.js

```diff
@@ -0,0 +1,80 @@
+Ext.namespace("GEOR");
+
+GEOR.custom = {
```

Comment:
I'll synchronize the GEOR_custom.js from template into the default debian package provided file. (Anyway, we will encounter the same issue as the one we just discussed with the addons ...)